backend/write_to_catalog/task.py

```diff
@@ -0,0 +1,57 @@
+import boto3
+from pystac import STAC_IO, Catalog, CatalogType  # type: ignore[import]
+
+from ..api_responses import BODY_KEY
+from ..pystac_io_methods import read_method, write_method
+from ..resources import ResourceName
+from ..types import JsonObject
+
+STAC_IO.write_text_method = write_method
+STAC_IO.read_text_method = read_method
+
+S3_CLIENT = boto3.client("s3")
+
+ROOT_CATALOG_ID = "root_catalog_id"
+ROOT_CATALOG_TITLE = "Geospatial Datalake Root Catalog"
+ROOT_CATALOG_DESCRIPTION = "The root catalog which links to all datasets catalogs in the data lake"
+CATALOG_KEY = "catalog.json"
+
+
+def lambda_handler(_event: JsonObject, _context: bytes) -> JsonObject:
```

Comment:
The underscore prefix in a parameter name is used to indicate (by convention only) that the argument value is unused, and is just there because whatever calls this code is out of our control. Since the event object is used it should probably not have an underscore prefix.